File: problemas/iniciante/1154/1154.scala
object Main {
  def main(args: Array[String]): Unit = {
    var soma = 0
    var quantidade = 0
    var t = io.StdIn.readLine().toInt
    while(t >= 0) {
      soma += t
      quantidade += 1
      t = io.StdIn.readLine().toInt
    }

    printf("%.2f\n", soma.toFloat / quantidade)
  }
}

File: problemas/iniciante/1154/1154.swift
import Foundation

var soma = 0
var quantidade = 0
while (true) {
  let idade = Int(readLine(strippingNewline: true)!)!
  if idade < 0 {
    break
  }
  soma += idade
  quantidade += 1
}

print(String(format: "%.2f", Double(soma) / Double(quantidade)))